src/payment/strategies/googlepay/googlepay-payment-processor.ts

```diff
@@ -197,4 +208,36 @@ export default class GooglePayPaymentProcessor {
             customFields: [],
         };
     }
+
+    private _postForm(postPaymentData: TokenizePayload): Promise<Response<any>> {
+        const cardInformation = postPaymentData.details;
+
+        return this._requestSender.post('/checkout.php', {
+            headers: {
+                Accept: 'text/html',
+                'Content-Type': 'application/x-www-form-urlencoded',
+            },
+            body: toFormUrlEncoded({
+                payment_type: postPaymentData.type,
+                nonce: postPaymentData.nonce,
+                provider: this._getMethodId(),
+                action: 'set_external_checkout',
+                card_information: this._getCardInformation(cardInformation),
+            }),
+        });
+    }
+
+    private _updateBillingAddress(paymentData: GooglePaymentData): Promise<InternalCheckoutSelectors> {
+        const remoteBillingAddress = this._store.getState().billingAddress.getBillingAddress();
+
+        if (!remoteBillingAddress) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
```

Comment:
Do you really want to throw `MissingPaymentMethod` if billing address is undefined?

Reply:
Maybe `MissingCheckoutConfig?

Reply:
I think you need to create a new type, as config is not related to billing address data.